File: 0010-0019/0011.py
w = int(raw_input())
n = int(raw_input())

arr = range(1, w+1)
for i in range(n):
  a, b = map(int, raw_input().split(','))
  arr[a-1], arr[b-1] = arr[b-1], arr[a-1]

for a in arr:
  print '%d'%a

File: ruby/0011.rb
w = gets.to_i
n = gets.to_i

arr = (1..w).to_a
while str = gets
  a, b = str.split(',').map(&:to_i)
  arr[a-1], arr[b-1] = arr[b-1], arr[a-1]
end

puts arr
